src/functions/users/flextime/handler.ts

```diff
@@ -32,34 +22,57 @@ export const listUsersFlextimeHandler: APIGatewayProxyHandler = async (event) =>
     const usersWithFlextime = await Promise.allSettled(
       optedInUsers.map(async (severaUser) => {
         try {
+          /**
+           * TODO: This should be typed according to the spec response type for the lambda function
+           * Spec in the lambdas for the futur
+           */
```

Comment:
As a note This type of comment with /** is a TSDOc and what we use for function comments. 
For general notes/ TODO we just add a single line comment starting with //
Please update